metron-platform/metron-solr/src/main/config/schema/error/schema.xml

```diff
@@ -40,6 +39,8 @@
   <field name="guid" type="string" indexed="true" stored="true" required="true" multiValued="false" />
   <uniqueKey>guid</uniqueKey>
 
+  <!-- Raw messages can be split into multiple fields -->
+  <dynamicField name="raw_message*" type="text_raw" indexed="false" stored="true"/>
```

Comment:
Out of curiosity, under which circumstances do we get multiple raw messages?

Reply:
When the BulkWriterComponent hits an error, there's aggregation of the messages of the Tuples involved.

See:
https://github.com/apache/metron/blob/f072ed231f9acdf0957f0244b6402b2ecd9403a0/metron-platform/metron-writer/src/main/java/org/apache/metron/writer/BulkWriterComponent.java#L117

And:
https://github.com/apache/metron/blob/358e40bd17d5de2a380a91115eaee9ebd3a453f6/metron-platform/metron-common/src/main/java/org/apache/metron/common/error/MetronError.java#L147

Reply:
Makes sense, +1 on this then.